packages/flutter/lib/src/widgets/slotted_render_object_widget.dart

```diff
@@ -39,9 +44,25 @@ import 'framework.dart';
 ///
 ///   * [MultiChildRenderObjectWidget], which configures a [RenderObject]
 ///     with a single list of children.
-///   * [ListTile], which uses [SlottedMultiChildRenderObjectWidgetMixin] in its
+///   * [ListTile], which uses [SlottedMultiChildRenderObjectWidget] in its
 ///     internal (private) implementation.
-mixin SlottedMultiChildRenderObjectWidgetMixin<S> on RenderObjectWidget {
+abstract class SlottedMultiChildRenderObjectWidget<S, T extends RenderObject> extends RenderObjectWidget with SlottedMultiChildRenderObjectWidgetMixin<S, T> {
```

Comment:
Should we change S and T to `SlotType` and `ChildType` to be a little more self explanatory? I believe, we use `ChildType` for this elsewhere in the framework.

Reply:
done